src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(input) {
+    if (!input && input !== 0) {
```

Comment:
I think it's better to do it this way
```suggestion
    if (input === undefined) {
```